go.mod

```diff
@@ -13,15 +13,15 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/websocket v1.5.3
 	github.com/mattn/go-colorable v0.1.13
-	github.com/onsi/ginkgo/v2 v2.4.0
-	github.com/onsi/gomega v1.24.0
+	github.com/onsi/ginkgo/v2 v2.22.2
+	github.com/onsi/gomega v1.36.2
 	github.com/pebbe/zmq4 v1.2.10
 	github.com/prometheus/client_golang v1.14.0
 	github.com/sirupsen/logrus v1.9.0
 	github.com/smira/go-statsd v1.3.2
 	go.uber.org/automaxprocs v1.5.2
 	google.golang.org/api v0.114.0
-	google.golang.org/protobuf v1.35.1
+	google.golang.org/protobuf v1.36.1
```

Comment:
This was introduced when upgrade ginkgo and gomega as it is a dependency https://github.com/onsi/ginkgo/blob/v2.22.2/go.mod#L20